wp-content/mu-plugins/the-world-site-config/configs/development/development-plugins.php

```diff
@@ -5,4 +5,16 @@
  * @package the_world_site_config
  */
 
-define( 'DEVELOPMENT_PLUGINS', array() );
+define(
+	'DEVELOPMENT_PLUGINS',
+	array(
+		'fg-drupal-to-wp-premium/fg-drupal-to-wp-premium.php',
+		'fg-drupal-to-wp-premium-entityreference-module/fg-drupal-to-wp-entityreference.php',
+		'fg-drupal-to-wp-premium-fieldcollection-module/fg-drupal-to-wp-fieldcollection.php',
+		'fg-drupal-to-wp-premium-mediaprovider-module/fg-drupal-to-wp-mediaprovider.php',
+		'fg-drupal-to-wp-premium-metatag-module/fg-drupal-to-wp-metatag.php',
+		'pri-migration-helper/pri-migration-helper.php',
+		'pri-external-attachment/pri-external-attachment.php',
+	)
```

Comment:
Should probably move these to the global plugins if we plan to run this in live environment.

Reply:
You Are right @rpeterman-gp i will do that and push with other changes in case you find something else.

Reply:
Plugins were added to Production env